evm/utils.js

```diff
@@ -501,12 +501,7 @@ const getEVMAddresses = async (config, chain, options = {}) => {
 };
 
 const getAmplifierKeyAddresses = async (config, chain) => {
```

Comment:
```suggestion
 const getAmplifierVerifiers = async (config, chain) => {
```
For consistency everywhere?

Reply:
updated